### ISSUE TITLE
Store vector embeddings

### ISSUE THREAD

**Issue by mihow** [RolnickLab/antenna#752]
## Summary

Add support for saving vector embeddings from model output to be used for unknown species detection & similarity search.

## Implementation Details

We can begin by returning and storing vectors for each classification for each detection in the same way we are returning & storing logits in the postgres database and querying with pgvector. But consider exploring adding a new dedicated vector database to the stack. 

Need support for multiple embeddings! but is one per each classification/model output okay? When we get to querying/using the vectors, that is the point when we need select which embedding (or the model that made them if its one-to-one)

## Related Issues

...

## Additional Context

...

**Comment by mihow** [comment]
Approaches to test:
- Add pgvector to our Antenna postgres database & add similarity search queries to the Django ORM

**Comment by mihow** [comment]
Closed by #821
